src/languages/objectivec.js

```diff
@@ -12,46 +12,51 @@ export default function(hljs) {
     begin: '\\b(AV|CA|CF|CG|CI|CL|CM|CN|CT|MK|MP|MTK|MTL|NS|SCN|SK|UI|WK|XC)\\w+'
   };
   const IDENTIFIER_RE = /[a-zA-Z@][a-zA-Z0-9_]*/;
-  const KWS = [
+  const TYPES = [
     "int",
     "float",
-    "while",
     "char",
+    "unsigned",
+    "signed",
+    "short",
+    "long",
+    "double",
+    "wchar_t",
+    "unichar",
+    "void",
+    "bool",
+    "BOOL",
+    "id|0",
```

Comment:
What's this? I can't find any documentation on what `id|0` means in obj-c.

Reply:
<https://www.google.com/search?client=safari&rls=en&q=objective+c+types+%22id%22&ie=UTF-8&oe=UTF-8>

Just what google told me...

Reply:
Ah I was looking for the `id|0` as a whole and couldn't find anything. What does the `|0` mean?

Reply:
0 relevance.

Reply:
🤦  I didn't realize that was shorthand for relevance